examples/example_acmpca.rb

```diff
@@ -0,0 +1,30 @@
+## Run this from the root directory of geo.
+##
+##  ./bin/geo plan examples/example_acmpca.rb
+##
+GeoCLI.instance.env_name = "geoexperiments"
+GeoCLI.instance.environment = GeoEngineer::Environment.new("geoexperiments")
+project = GeoCLI.instance.environment.project("org","geoexperiments"){
+    environments "geoexperiments"
+}
+
+
+acmpca = project.resource("aws_acmpca_certificate_authority", "acbhq_dot_net") {
+    certificate_authority_configuration {
+       key_algorithm      "RSA_4096"
+       signing_algorithm "SHA512WITHRSA"
+
+       subject {
+         common_name  "acbhq.net"
+         country "US"
+         state "California"
+         organization "ABCCompany"
+       }
+     }
+    type "ROOT"
+    tags {
+      name "acbhq.net"
+    }
+  }
+
+p acmpca
```

Comment:
I really like the idea of working examples. What do you think about examples that were part of our automated testing process so that we know that the code is up to date and works?

As written, I'm wondering how we ensure these are discoverable and up to date.

Reply:
One way I do this is through Makefiles. Tests are great, but they don't surface the issues that can occur when running a geo plan. I can check this in if it is useful to others and we can iterate over the makefile

```
run-example-cert:
	./bin/geo plan examples/example_acmpca_certificate.rb

run-example-acmpca:
	./bin/geo plan examples/example_acmpca.rb
```